storage/tests/system.py

```diff
@@ -179,6 +179,27 @@ def test_crud_bucket_with_requester_pays(self):
             # Exercise 'buckets.delete' w/ userProject.
             with_up.delete()
 
+    @unittest.skipUnless(USER_PROJECT, 'USER_PROJECT not set in environment.')
+    def test_bucket_get_blob_with_user_project(self):
+        new_bucket_name = 'w-requester-pays' + unique_resource_id('-')
+        created = Config.CLIENT.create_bucket(
+            new_bucket_name, requester_pays=True)
+        self.case_buckets_to_delete.append(new_bucket_name)
+        self.assertEqual(created.name, new_bucket_name)
+        self.assertTrue(created.requester_pays)
+
+        with_user_project = Config.CLIENT.bucket(
+            new_bucket_name, user_project=USER_PROJECT)
+
+        self.assertIsNone(with_user_project.get_blob('nonesuch'))
+        to_add = created.blob('blob-name')
+        to_add.upload_from_string(b'DEADBEEF')
```

Comment:
Suggest: `data = b'DEADBEEF'`, then you won't need to copy it below.

As before, `try / finally` breaks from our usual clean-up. (It also breaks from Python's "one way to do it" mantra, since now we cleanup in system tests in two diff. ways.)

Reply:
> Suggest: data = b'DEADBEEF', then you won't need to copy it below.

244ff74

> `try / finally` breaks from our usual clean-up. (It also breaks from Python's "one way to do it" mantra, since now we cleanup in system tests in two diff. ways.)

Here, the 'try: finally:' is to deal with a blob added in a "bucket-centric" test, whose `tearDown` does not know from blobs (the storage system tests are factored somewhat oddly).

Reply:
> the storage system tests are factored somewhat oddly

So change them?